.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+name: "NodeJs Init Container CI"
+
+on:
+  pull_request:
+    paths:
+      - 'src/nodejs/**'
+      - '.github/workflows/nodejs.yml'
+  push:
+    paths:
+      - 'nodejs/**'
+      - '.github/workflows/nodejs.yml'
+    branches:
+      - main
+    # Do not run when a tag is created.
+    tags-ignore:
+      - "**"
+  release:
+    types:
+      - published
+  workflow_dispatch:
+
+env:
+  INITCONTAINER_LANGUAGE: nodejs
+  K8S_OPERATOR_IMAGE_TAG: edge
+
+jobs:
+  call-test:
+    if: github.event_name == 'release' && endsWith(github.ref, '_nodejs') # Skip everything if this isn't Node
+    uses: ./.github/workflows/test.yml
+    with:
+      RUNTIME_VERSION: ${{ matrix.node-version }}
+
+  call-publish:
+    runs-on: ubuntu-latest
+    needs:
+      - call-test
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16, 18, 20]
+        agent-version: ['latest']
+
+    steps:
+      - uses: actions/setup-node@v4
+        with:
+          INITCONTAINER_LANGUAGE: ${{ env.INITCONTAINER_LANGUAGE }}
+          K8S_OPERATOR_IMAGE_TAG: ${{ env.K8S_OPERATOR_IMAGE_TAG }}
+          RUNTIME_VERSION: ${{ matrix.node-version }}
+
+      - name: Get agent version
+        run: |
+          if [[ ${{ matrix.agent-version }} == 'latest' ]]; then
+            cd nodejs && npm i newrelic@${{ matrix.agent-version }} && cd ..
+            echo AGENT_VERSION=$(cat nodejs/package.json | jq -r '.dependencies."newrelic"') >> $GITHUB_ENV 
+          elif 
```

Comment:
should this be `else`? (I'm no bash expert...)

Reply:
`elif` is correct for bash.